packages/storybook/src/nimble/tabs/tabs-matrix.stories.ts

```diff
@@ -45,14 +50,16 @@ const component = (
             Tab Two ${() => disabledName}
         </${tabTag}>
         <${tabTag} hidden>Tab Three</${tabTag}>
```

Comment:
If I go in devtools and change the tab from hidden to visible then operate the left right arrows I get an error:
![arrows-after-hidden](https://github.com/user-attachments/assets/9f405722-a8b9-4e1a-b890-652dff1f2490)